test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/CompositeModelBinderTest.cs

```diff
@@ -547,26 +526,6 @@ private static CompositeModelBinder CreateCompositeBinder(IModelBinder mockIntBi
             return shimBinder;
         }
 
-        private static IModelValidatorProvider GetValidatorProvider(params IModelValidator[] validators)
```

Comment:
Yah mean /none/ of the tests here validated? That makes sense since this is a unit test but still surprising this code was ever written. Hopefully was just over-zealous copying 'n pasting :smile: